components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsAuthenticatedUser.java

```diff
@@ -37,10 +39,17 @@
  */
 public class JsAuthenticatedUser extends AbstractJSObjectWrapper<AuthenticatedUser> {
 
-    public JsAuthenticatedUser(AuthenticatedUser wrapped) {
+    private AuthenticationContext context;
+    private int step;
+    private String idp;
+
+    public JsAuthenticatedUser(AuthenticatedUser wrapped, AuthenticationContext context, int step, String idp) {
```

Comment:
Shouldn't we also have another constructor which simply wraps the AuthenticatedUser object only?

Reply:
+1, will add